platform/ext/target/nordic_nrf/common/core/cmsis_drivers/Driver_USART.c

```diff
@@ -30,6 +33,43 @@
 
 #if RTE_USART0 || RTE_USART1 || RTE_USART2 || RTE_USART3
 
+#define PSEL_DISCONNECTED 0xFFFFFFFFUL
+
+#define UART_CONFIG_INITIALIZER()                                              \
+{                                                                              \
+    .txd_pin  = PSEL_DISCONNECTED,                                             \
+    .rxd_pin  = PSEL_DISCONNECTED,                                             \
+    .rts_pin  = PSEL_DISCONNECTED,                                             \
+    .cts_pin  = PSEL_DISCONNECTED,                                             \
+    .baudrate = NRF_UARTE_BAUDRATE_115200,                                     \
+    .interrupt_priority = NRFX_UARTE_DEFAULT_CONFIG_IRQ_PRIORITY,              \
+    .config  = {                                                               \
+        .hwfc   = NRF_UARTE_HWFC_DISABLED,                                     \
+        .parity = NRF_UARTE_PARITY_EXCLUDED,                                   \
+        .stop   = NRF_UARTE_STOP_ONE,                                          \
+    },                                                                         \
+}
+
+void uart_config_set_uart_pins(nrfx_uarte_config_t *uart_config,
```

Comment:
```suggestion
static void uart_config_set_uart_pins(nrfx_uarte_config_t *uart_config,
```

Reply:
@anangl I don't disagree, but these patches are already merged in upstream TF-M now.
I will submit a patch upstream for this, but I think we can skip this change until next sync with TF-M.